app/models/attachment.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Attachment < ApplicationRecord
+  has_one_attached :media_file
+
+  validates :name, presence: true
+  validates :media_file, attached: true, content_type: %i[png jpg jpeg]
```

Comment:
not critical, but i think adding max_size validation will fit good here

Reply:
And for the :name also would be great to add min && max length validation.

Reply:
fixed